opt-may-2016/krishna/com/two95/training/Inter/ToyotaDealer.java

```diff
@@ -0,0 +1,6 @@
+package com.two95.training.Inter;
+public class ToyotaDealer extends Toyota {

```

Comment:
Toyota Dealer will not extend Toyota..but instead it will associate Toyota..it will just have a method public void sellToyotacar(Toyota car){}